engine/execution/computation/computer/computer.go

```diff
@@ -239,11 +239,19 @@ func (e *blockComputer) executeSystemCollection(
 		return txIndex, fmt.Errorf("could not get system chunk transaction: %w", err)
 	}
 
-	err = e.executeTransaction(tx, colSpan, collectionView, programs, systemChunkCtx, collectionIndex, txIndex, res)
+	err, txerr := e.executeTransaction(tx, colSpan, collectionView, programs, systemChunkCtx, collectionIndex, txIndex, res)
```

Comment:
I would oppose this change. 
Transaction error is not an error in terms of execution - its successful execution from EN code point of view. We leak the logic here.
Tx result is available for inspection inside `res` here.

Reply:
I misled them. I was worried that looking at the last item in res might not be trivial and would be based on assumption that this tx is always the last one, so though wiring the txError upward might be okey.